src/ambient.d.ts

```diff
@@ -10,10 +10,6 @@ declare global {
       appPaths: Record<string, string>;
       app: AppType;
       contentChangeListeners: Array<any>;
-      editors: Record<
```

Comment:
Looks like the imports got left around - we should remove:

```ts
import * as MonacoType from 'monaco-editor';
```
and
```ts
import { EditorId } from './interfaces';
```

Reply:
Fixed in 306573e19585ce66bf824cd4ae75ae3024367d9d